From 22ffed1223fbf8076ad7c72a9fecb51005a26251 Mon Sep 17 00:00:00 2001
From: Evgeni Golov <evgeni@golov.de>
Date: Tue, 12 Mar 2013 08:07:36 +0100
Subject: [PATCH] drop dnsbl.njabl.org, it's not maintained anymore

---
 bley.conf |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bley.conf b/bley.conf
index 21de6ae..c20e7bd 100644
--- a/bley.conf
+++ b/bley.conf
@@ -18,7 +18,7 @@ dbpass = bley
 dbname = bley
 
 # Which DNSBLs and DNSWLs to use?
-dnsbls = ix.dnsbl.manitu.net, dnsbl.njabl.org, dnsbl.ahbl.org, dnsbl.sorbs.net
+dnsbls = ix.dnsbl.manitu.net, dnsbl.ahbl.org, dnsbl.sorbs.net
 dnswls = list.dnswl.org
 
 # Whitelist after dnswl_threshold hits.
-- 
1.7.10.4

