From e4eddec83e6e2337d5ab57d2ff1f2979ec412f81 Mon Sep 17 00:00:00 2001
From: Evgeni Golov <evgeni@golov.de>
Date: Wed, 7 Jan 2015 21:55:14 +0100
Subject: [PATCH] drop dnsbl.ahbl.org from the config example, it's dead

---
 bley.conf.example | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bley.conf.example b/bley.conf.example
index 2a133b9..d0786ad 100644
--- a/bley.conf.example
+++ b/bley.conf.example
@@ -24,7 +24,7 @@ whitelist_recipients_file = ./whitelist_recipients
 whitelist_clients_file = ./whitelist_clients
 
 # Which DNSBLs and DNSWLs to use?
-dnsbls = ix.dnsbl.manitu.net, dnsbl.ahbl.org, dnsbl.sorbs.net
+dnsbls = ix.dnsbl.manitu.net, dnsbl.sorbs.net
 dnswls = list.dnswl.org
 
 # Whitelist after dnswl_threshold hits.
-- 
2.1.4

